docs/samples/kafka/image_transformer/__main__.py

```diff
@@ -27,4 +28,6 @@
 if __name__ == "__main__":
     transformer = ImageTransformer(args.model_name, predictor_host=args.predictor_host)
     server = kserve.ModelServer()
-    server.start(models=[transformer])
+    asyncio.run(
```

Comment:
This seems like a breaking change for user, can we do `asyncio` inside `server.start`?